python/ray/serve/tests/test_config_files/logging_config_test.py

```diff
@@ -34,6 +36,14 @@ async def __call__(self):
         else:
             log_info["router_log_file"] = None
         log_info["router_log_level"] = logger.level
+
+        try:
+            # Add controller log file path
+            client = _get_global_client()
+            _, log_file_path = ray.get(client._controller._get_logging_config.remote())
+        except Exception:
```

Comment:
Non-blocker: would be nice to catch the specific exception if we know what it is

Reply:
+1, we should catch the specific exception here.